inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_strided_slice.cpp

```diff
@@ -132,8 +133,6 @@ void dynamicToStaticShapeStridedSlice(std::shared_ptr<ngraph::Node> target) {
     const auto all_zero = [](const std::vector<int64_t> & v) {return std::all_of(v.cbegin(), v.cend(), [](const int64_t & i){return i == 0;});};
     VPU_THROW_UNLESS(all_zero(stridedSlice->get_new_axis_mask()),
             "dynamicToStaticShapeStridedSlice transformation is not applicable for {}, new_axis_mask expected to be zeros", target);
-    VPU_THROW_UNLESS(all_zero(stridedSlice->get_shrink_axis_mask()),
-                "dynamicToStaticShapeStridedSlice transformation is not applicable for {}, shrink_axis_mask expected to be zeros", target);
     VPU_THROW_UNLESS(all_zero(stridedSlice->get_ellipsis_mask()),
```

Comment:
AFAIK, new_axis_mask and ellipsis_mask are also supported by Myriad plugin. 
Will it be expensive to support them in DTS as well?

Reply:
Discussed offline. Resolve